en/docs/setup/migrating-preparing-for-migration.md

```diff
@@ -21,6 +21,113 @@
 
 With WSO2 Identity Server 5.11.0, groups and roles are separated. For more information, see [What Has Changed in 5.11.0](../../setup/migrating-what-has-changed#group-and-role-separation).
 
+## Groups and Roles Improvements Migration
+
+With WSO2 Identity Server 5.12.0, groups and roles improvements are introduced. For more information, see [What Has Changed in 5.11.0](../../setup/migrating-what-has-changed#roup and role separation Improvements).
```

Comment:
shouldn't this be 5.11.0 instead of 5.12.0?

Reply:
Through separation introduced in 5.11.0, `separation improvements` are introduced with 5.12.0